app/src/main/java/com/tari/android/wallet/ui/activity/CreateWalletActivity.kt

```diff
@@ -0,0 +1,208 @@
+package com.tari.android.wallet.ui.activity
+
+import android.animation.*
+import android.os.Bundle
+import android.os.Handler
+import android.view.View
+import android.view.ViewTreeObserver
+import android.widget.*
+import androidx.core.view.marginTop
+import butterknife.BindDimen
+import butterknife.BindView
+import butterknife.OnClick
+import com.airbnb.lottie.LottieAnimationView
+import com.daasuu.ei.Ease
+import com.daasuu.ei.EasingInterpolator
+import com.tari.android.wallet.R
+import com.tari.android.wallet.ui.util.UiUtil.getResourceUri
+import com.tari.android.wallet.util.Constants
+
+import java.lang.ref.WeakReference
+
+class CreateWalletActivity : BaseActivity() {
+
+    @BindView(R.id.main_anim_tari)
+    lateinit var tariAnimationView: LottieAnimationView
+    @BindView(R.id.rootView)
+    lateinit var rootView: FrameLayout
+    @BindView(R.id.crypto_wallet_main_txt1)
+    lateinit var cryptoWalletMainText1: TextView
```

Comment:
Is this the first line of a 2-line text? If so cryptoWalletMainTextLine1 or something indicating the function better would preferable.